suites/pacific/rgw/tier-1_longevity.yaml

```diff
@@ -0,0 +1,253 @@
+                        - mero011
+                        - mero012
+                        - mero013
+                        - mero014
+                        - mero015
+                        - mero016
+
+              - config:
+                  command: apply
+                  service: rgw
+                  pos_args:
+                    - shared.sec.8081
+                  args:
+                    port: 8081
+                    placement:
+                      nodes:
+                        - mero010
+                        - mero011
+                        - mero012
+                        - mero013
+                        - mero014
+                        - mero015
+                        - mero016
+
+      desc: RHCS cluster deployment using cephadm.
+      destroy-cluster: false
+      module: test_cephadm.py
+      name: deploy cluster
+
+  - test:
+      abort-on-fail: true
+      clusters:
+        ceph-pri:
+          config:
+            command: add
+            id: client.pri
+            node: extensa013
+            install_packages:
+              - ceph-common
+            copy_admin_keyring: true
+        ceph-sec:
+          config:
+            command: add
+            id: client.sec
+            node: extensa016
+            install_packages:
+              - ceph-common
+            copy_admin_keyring: true
+      desc: Configure the RGW client system
+      destroy-cluster: false
+      module: test_client.py
+      name: configure client
+
+  - test:
+      abort-on-fail: true
+      clusters:
+        ceph-pri:
+          config:
+            cephadm: true
+            commands:
+              - "radosgw-admin realm create --rgw-realm india --default"
+              - "radosgw-admin zonegroup create --rgw-realm india --rgw-zonegroup shared --endpoints http://{node_ip:mero001}:8080,http://{node_ip:mero007}:8080,http://{node_ip:mero003}:8080,http://{node_ip:mero004}:8080 --master --default"
+              - "radosgw-admin zone create --rgw-realm india --rgw-zonegroup shared --rgw-zone primary --endpoints http://{node_ip:mero001}:8080,http://{node_ip:mero007}:8080,http://{node_ip:mero003}:8080,http://{node_ip:mero004}:8080 --master --default"
+              - "radosgw-admin period update --rgw-realm india --commit"
+              - "radosgw-admin user create --uid=repuser --display_name='Replication user' --access-key test123 --secret test123 --rgw-realm india --system"
+              - "radosgw-admin zone modify --rgw-realm india --rgw-zonegroup shared --rgw-zone primary --access-key test123 --secret test123"
+              - "radosgw-admin period update --rgw-realm india --commit"
+              - "ceph config set client.rgw.* rgw_realm india"
+              - "ceph config set client.rgw.* rgw_zonegroup shared"
+              - "ceph config set client.rgw.* rgw_zone primary"
+              - "ceph orch restart rgw.shared.pri.8080"
+              - "ceph orch restart rgw.shared.pri.8081"
```

Comment:
Would it make sense to have a module that would help us perform the multi-site configurations?

Reply:
I think for bare-metal rgw-multisite deployment we would need to hardcode multisite sync endpoints as per the standard configuration.

Reply:
Let's pick this in a different PR